app/controllers/highlights_controller.rb

```diff
@@ -0,0 +1,74 @@
+class HighlightsController < ApplicationController
+  before_action :set_highlight, only: [:show, :edit, :update, :destroy]
+
+  # GET /highlights
+  # GET /highlights.json
+  def index
+    @highlights = Highlight.all
+  end
+
+  # GET /highlights/1
+  # GET /highlights/1.json
+  def show
+  end
+
+  # GET /highlights/new
+  def new
+    @highlight = Highlight.new
+  end
+
+  # GET /highlights/1/edit
+  def edit
+  end
+
+  # POST /highlights
+  # POST /highlights.json
+  def create
+    @highlight = Highlight.new(highlight_params)
+
+    respond_to do |format|
+      if @highlight.save
+        format.html { redirect_to @highlight, notice: 'Highlight was successfully created.' }
+        format.json { render :show, status: :created, location: @highlight }
+      else
+        format.html { render :new }
+        format.json { render json: @highlight.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /highlights/1
+  # PATCH/PUT /highlights/1.json
+  def update
+    respond_to do |format|
+      if @highlight.update(highlight_params)
+        format.html { redirect_to @highlight, notice: 'Highlight was successfully updated.' }
+        format.json { render :show, status: :ok, location: @highlight }
+      else
+        format.html { render :edit }
+        format.json { render json: @highlight.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /highlights/1
+  # DELETE /highlights/1.json
+  def destroy
```

Comment:
Do we want users to be able to destroy the highlights? Maybe this shouldn't be here